docs/Reference/API-Objects.md

```diff
@@ -254,11 +254,20 @@ Returned by [`trace_replayBlockTransactions`](API-Methods.md#trace_replayblocktr
 
 Returned by [`priv_getTransactionReceipt`](API-Methods.md#priv_getTransactionReceipt).
 
-| Key | Type | Value |
-|-----|:----:|-------|
-| **contractAddress** | Data, 20&nbsp;bytes | Contract address created if a contract creation transaction, otherwise, `null`.    |
-| **from**            | Data, 20&nbsp;bytes | Address of the sender.                                                             |
-| **output**          | Data                | RLP-encoded return value of a contract call if a value returns, otherwise, `null`. |
-| **commitmentHash**  | Data, 32&nbsp;bytes | Hash of the privacy marker transaction.                                            |
-| **transactionHash** | Data, 32&nbsp;bytes | Hash of the private transaction.                                                   |
-| **privateFrom**     | Data, 32&nbsp;bytes | [Orion](https://docs.orion.consensys.net/en/stable/) public key of the sender.     |
+| Key                                  | Type                         | Value                                                                                                  |
```

Comment:
Could you try to keep the table readable also in the source code so it eases the reviews please?
It will require to adjust some lines manually to display correctly but the goal is source code readability, correct display in MkDocs site and in Github preview.